script.js

```diff
@@ -1 +1,19 @@
+const todoInput = document.getElementById("todo-input");
+const addTodoBtn = document.getElementById("add-todo-btn");
+const taskListContainer = document.getElementById("task-list-container");
+const taskList = document.getElementById("task-list");
 
+addTodoBtn.addEventListener("click", addTodo);
+
+/**
+ * Adding todo, clearing input after adding
+ */
```

Comment:
While comments can be helpful for future maintainers/collaborators, it's often a good practice to leave them out. This is also known as code smell. 
Ideally, your variable naming, function naming and code flow should be self describing. 

Let's remove this comment.
```suggestion
```